cosmic-metrics-collector/src/main/resources/queries.yml

```diff
@@ -16,3 +16,16 @@ virtual-machine-metrics-repository:
       vm.state IN ('Running', 'Stopped')
     AND
       vm.removed IS NULL
+
+public-ip-address-metrics-repository:
+  get-metrics: |
+    SELECT
```

Comment:
We should filter by `state` as well. If there's a problem while the public IP address is being allocated (with `Allocating` state), we shouldn't bill that IP. So, please add to the `WHERE` clause: `ip.state = 'Allocated'`